src/mumble/Themes.cpp

```diff
@@ -55,25 +63,59 @@ void Themes::applyFallback() {
 }
 
 bool Themes::applyConfigured() {
+	static QString currentThemePath;
```

Comment:
Please use a member in the Themes class or a string in the Mumble setting or something, as opposed to this ``static`` construct.